src/utils/LyricsUtils.js

```diff
@@ -0,0 +1,18 @@
+const snekfetch = require('snekfetch')

```

Comment:
This should actually be inside of the APIs folder. APIs only load if they have the required env variables, and the commands that need them only load if they've loaded.